develop/TOOLS/DIS/Guides/DisMibBrowser.md

```diff
@@ -0,0 +1,118 @@
+> This will prevent table names from being included twice in column parameter names.
+
+### Bulk Parameter Generation
+
+For larger projects, generating multiple parameters concurrently becomes invaluable:
+
+- Right-click on your desired location within the XML.
+- From the context menu, choose the *Generate Parameters...* option.
+
+![DIS ToolWindow - MIB Browser - ContextMenu - Generate Parameters](~/develop/images/DIS_ToolWindow_MibBrowser_ContextMenu_GenerateParameters.png)
+
+#### Generate Parameters Wizard
+
+1. **Select Input Type**: By default, *MIB* is preselected. Click *Next* to proceed.
+1. **Choose Desired Parameters**: View the items present in the MIB Browser and select the ones you wish to generate. For table generation, the wizard automatically adds the columns, but you can also omit unnecessary columns. After making your selections, press *Next*.
+
+   ![DIS - Generate Parameters - Choose Params](~/develop/images/DIS_ToolWindow_MibBrowser_GenerateParameters_ChooseParams.png)
+
+1. **Set Initial Parameter ID**: Define a starting parameter ID for the generated parameters. Adjust the offset for write parameters. Then, proceed with *Next*.
+1. **Check ID Validity**: This screen previews the upcoming parameter generation, highlighting any potential ID conflicts. If everything looks in order, click *Next*. If you wish to bypass additional settings and generate parameters immediately, select *Finish*.
+
+   ![DIS - Generate Parameters - Validate IDs](~/develop/images/DIS_ToolWindow_MibBrowser_GenerateParameters_ValidateIds.png)
+
+1. **Manage Timers**: View and modify current timers in the connector, or introduce new ones. Opt for *Next* to define groups or *Finish* to generate parameters and timers.
+
+   ![DIS - Generate Parameters - Timers](~/develop/images/DIS_ToolWindow_MibBrowser_GenerateParameters_Timers.png)
+
+1. **Establish or Update Groups**: The wizard offers group suggestions based on your current selections. You can edit these or introduce new ones. Assign each group to its appropriate timer if applicable, then click *Next*.
+
+   ![DIS - Generate Parameters - Groups](~/develop/images/DIS_ToolWindow_MibBrowser_GenerateParameters_Groups.png)
+
+1. **Group Parameter Assignment**: Here you can decide which group each parameter belongs to. Once satisfied with your configurations, hit *Finish*. Your parameters, alongside their respective groups and timers, will then be generated.
+
+   ![DIS - Generate Parameters - Group Params Assignment](~/develop/images/DIS_ToolWindow_MibBrowser_GenerateParameters_GroupParams.png)
+
+## Comparing Connector and MIB Files
+
+The ability to swiftly identify descrepancies between your connector and MIB files is crucial. The DIS MIB Browser provides an efficient solution for this via its *Compare* tab. This tool helps ensure that your SNMP connectors are always updated in line with the most recent MIB files.
```

Comment:
I would put the 'why' first. Proposal for a rewrite: 
"It's crucial to ensure that your SNMP connectors are up-to-date with the latest MIB files. The DIS MIB Browser's *Compare* tab offers an effective solution to quickly identify discrepancies between your connector and the MIB files."